### PR TITLE
13 add logic to parsing to handle event name event description

### LINKED ISSUE #13: Add logic to parsing to handle event name, event description
- Context for meeting
- make it more fun
- Have different type of meeting options for 2 ppl vs 2+ ppl (e.g. pairing/coffee)

### PR DESCRIPTION
**Pull request by alexrosenblatt**
Initial check-in of meeting title handling. Not finished and needs better error handling.

### CODE REVIEW

File: zulip_bots/zulip_bots/bots/calendarbot/calendarbot.py [hunk 1]
```diff
@@ -1,32 +1,45 @@
 import logging
-from typing import Any, Dict, List, Optional, Union
```
Comment (alexrosenblatt) [inline zulip_bots/zulip_bots/bots/calendarbot/calendarbot.py:2]: This is just me sorting the imports.

File: calendarbot.log [hunk 2]
```diff
@@ -0,0 +1,53 @@
+DEBUG:root:Cred file cannot be loaded.
```
Comment (mwen00) [inline calendarbot.log:1]: Add this file to `.gitignore` maybe?

File: icebreakers.txt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Was there ever a food you didn’t want to eat, but you loved it once you tried it?
+If you had to live in a video game, which one would it be?
+If you could remove one word from the dictionary, what would it be?
+If you could be any animal, which would you be?
+If you could control one element (air, water, earth, fire), which would it be and why?
+If you could send a message out to the entire world, what would it be?
+If you could only watch one movie for the rest of your life, what would it be?
+If you could only listen to one album for the rest of your life, which would you choose?
+What’s your pet peeve?
+
```
Comment (mwen00) [inline icebreakers.txt:91]: lolololol some of these are so silly

**Review by mwen00 — APPROVED** [review]
These all look good to merge in! 😃

**Pull request merged**